src/centos/scripts/start-marklogic.sh

```diff
@@ -280,15 +281,21 @@ elif [[ "${MARKLOGIC_JOIN_CLUSTER}" == "true" ]] && [[ "${HOSTNAME}" != "${MARKL
         info "MARKLOGIC_GROUP is not specified, adding host to the Default group."
         MARKLOGIC_GROUP_PAYLOAD=\"group=Default\"
     else
-        info "MARKLOGIC_GROUP is specified, adding host to the ${MARKLOGIC_GROUP} group."
-        MARKLOGIC_GROUP_PAYLOAD=\"group=${MARKLOGIC_GROUP}\"
+        ML_GROUPS=$(curl --anyauth --user "${ML_ADMIN_USERNAME}":"${ML_ADMIN_PASSWORD}" -m 20 -s -X GET -H "Content-type:application/json" http://"${MARKLOGIC_BOOTSTRAP_HOST}":8002/manage/v2/groups?format=json |
```

Comment:
Can we just send a request like ${MARKLOGIC_BOOTSTRAP_HOST}:8002/manage/v2/groups/${GROUP_NAME} to check the existence of a group? if the group exist, the response code is 200, otherwise the response code is 404 with the message: XDMP-NOSUCHGROUP: xdmp:group("cnode") -- No such group cnode

Reply:
Agreed, changed the API